src/modelspec/utils.py

```diff
@@ -49,6 +61,13 @@ def save_to_yaml_file(info_dict, filename, indent=4):
         fp.write(stry)
 
 
+def save_to_bson_file(info_dict, filename):
```

Comment:
This function needs a docstring and type annotations. It seems like dead code though. I think a lot of these functions aren't used anymore so I don't think we need to continue adding functions here. All this functionality is present on the `Base` class.